index.html

```diff
@@ -308,7 +308,7 @@ <h4>Terminology</h4>
 &lt;/ul&gt;</code></pre>
                 </details></div>
             </li>
-            <li id="step2D">Otherwise, if the <code>current node</code>'s native markup provides an <a class="termref">attribute</a> (e.g. <code>title</code>) or <a class="termref">element</a> (e.g. HTML <code>label</code>) that defines a text alternative, return that alternative in the form of a <code>flat string</code> as defined by the host language, unless the element is marked as presentational (<code>role="presentation"</code> or <code>role="none"</code>).
+            <li id="step2D">Otherwise, if the <code>current node</code>'s native markup provides an <a class="termref">attribute</a> (e.g. <code>title</code>) or <a class="termref">element</a> (e.g. HTML <code>label</code>) that defines a text alternative, return that alternative in the form of a <code>flat string</code> as defined by the host language, unless the element is marked as presentational (<code>role="presentation"</code> or <code>role="none"</code>). See <a href="https://www.w3.org/TR/html-aam/">HTML-AAM</a> for detail on HTML markup that defines a text alternative.
```

Comment:
Is there a specific section we can link to? Otherwise considering a full spec as "details" might be a bit overwhelming. Maybe
```suggestion
            <li id="step2D">Otherwise, if the <code>current node</code>'s native markup provides an <a class="termref">attribute</a> (e.g. <code>title</code>) or <a class="termref">element</a> (e.g. HTML <code>label</code>) that defines a text alternative, return that alternative in the form of a <code>flat string</code> as defined by the host language, unless the element is marked as presentational (<code>role="presentation"</code> or <code>role="none"</code>). See <a href="https://www.w3.org/TR/html-aam/#accessible-name-and-description-computation">HTML-AAM</a> for detail on HTML markup that defines a text alternative.
```

This was suggested by @carmacleod  in https://github.com/w3c/accname/issues/36#issuecomment-436278363

Reply:
I agree, that section seems most relevant.